egs/ptb/LM/local/prepare_lm_training_data.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
```

Comment:
Shall we replace it with a symlink to `egs/librispeech/ASR/local/prepare_lm_training_data.py` ?